build.gradle

```diff
@@ -19,25 +19,21 @@ dependencies {
     testImplementation 'com.browserstack:browserstack-local-java:1.0.6'
     testImplementation 'org.apache.commons:commons-lang3:3.11'
     testImplementation 'org.slf4j:slf4j-api:1.7.30'
-    testImplementation 'ch.qos.logback:logback-classic:1.2.3'
```

Comment:
Why remove this library?

Reply:
i couldn't find the usage in the first level glance of the repo. okay to keep if this is being used.